README.md

```diff
@@ -632,6 +633,14 @@ let row = TextRow() {
 Swipe Actions need `tableView.isEditing` be set to `false`. Eureka will set this to `true` if there is a MultivaluedSection in the form (in the `viewWillAppear`).
 If you have both MultivaluedSections and swipe actions in the same form you should set `isEditing` according to your needs.
 
+## Custom form size
+
+Forms will span the entire view controller when you extend FormViewController. If you want to show your form in a subview of your view controller follow below steps:
+
+1. Add your UITableView to the view in your storyboard. Make sure the table view is styled as grouped rather than plain.
+2. Drag from the tableView circle in the Outlets panel in the Connections Inspector. This outlet is provided by Eureka's FormViewController so there is no need to override it or create a new @IBOutlet for the table view.
```

Comment:
I am  not convinced by the first sentence. I would say something like `Select the view controller and drag the tableView outlet from the Outlets panel to your UITableView on the storyboard.`